handlers.js

```diff
@@ -421,7 +421,7 @@ module.exports.addUserToTeam = async ({
       throw new HTTPError('Invalid access: team admin required', 403)
     }
 
-    let kvUser = await USERS.get(userEmail, { type: 'json' })
+    let kvUser = await USERS.get(userEmail.toLowerCase(), { type: 'json' })
```

Comment:
This would ignore issues retrospectively, such that if the KV already has the user's email in different cases, `USERS.get` will have a miss

Reply:
Yes, it will but that can be solved by manually making the ones already in USERS to lowercase?

Reply:
how would you resolve this issue in a backward compatible way, without the batch update in the existing deta base?

Reply:
No, that's what I meant to solve this a bulk update is required to the existing `USERS` and `users`.

Reply:
But there is one catch here, a bulk update to lowercase the email can be done but if there is a catch in the case where there are two entries for (say) `ashishjullia19@gmail.com` and `ASHISHJULLIA19@GMAIL.COM`, then if both have a few teams/stages/key:value(s) then this process can lead to a failure in such cases where either nit-pick the ones that can be discarded from the db means whether to keep `ashishjullia19@gmail.com` or `ASHISHJULLIA19@GMAIL.COM` (other making the email lowercase).